plugins/meta/portmap/README.md

```diff
@@ -0,0 +1,111 @@
+## Port-mapping plugin
+
+This plugin will forward traffic from one or more ports on the host to the
+container. It expects to be run as a chained plugin.
+
+## Usage
+You should use this plugin as part of a network configuration list. It accepts
+the following configuration options:
+
+* `noSnat` - boolean, default false. If true, do not set up the SNAT chains
```

Comment:
What are your thoughts on simply naming this field `snat` with a default true bool? Seems clearer than a double negative of `noSnat` and false.